cwms-data-api/src/main/webapp/times.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
this one probably should've gone in the docs folder. But again, better to have something now, we'll worry about it later.